app/models/project.rb

```diff
@@ -16,11 +16,13 @@
 
 class Project < ActiveRecord::Base
   include Sluggable
+  include ActiveModel::Validations
```

Comment:
I don't think you need to include this, `ActiveRecord::Base` already includes this AFAIK?